banner/javascript/sandpack.config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "files": [
+    "/index.html",
```

Comment:
not sure root relative file paths make sense

Reply:
these match sandpack configs and are appended using path so thought i'd try match sandpack config